spec.md

```diff
@@ -31,9 +31,21 @@ sequence of operations the program encapsulates.
 - `"format"`: Signifies the utilization of the PHIR/JSON format.
 - `"version"`: Represents the semantic version number adhering to the PHIR spec.
 - `"metadata"`: An optional segment where users can incorporate additional details. This segment holds potential for
-future expansion, possibly to guid compilation processes and error modeling.
+future expansion, possibly to guide compilation processes and error modeling.
 - `"ops": [{...}, ...]`: A linear sequence denoting the operations and blocks that constitute the program.
 
+## Metadata
+
+The metadata field takes several parameters that can be used to specify information relevant to the entire program
```

Comment:
This description feels somewhat inconsistent with the description above. We should mention that the following table shows possible metadata and should include other fields used in the examples in this spec.